src/Microsoft.ML.Data/DataLoadSave/DataLoadSaveCatalog.cs

```diff
@@ -2,19 +2,92 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using Microsoft.ML.Data;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.Internal.Utilities;
+using Microsoft.ML.Transforms;
+
 namespace Microsoft.ML.Runtime
 {
     /// <summary>
-    /// A catalog of operations to load and save data.
+    /// A catalog of operations over data that are not transformers or estimators.
+    /// This includes data readers, saving, caching, filtering etc.
     /// </summary>
-    public sealed class DataLoadSaveOperations
```

Comment:
>DataLoadSaveOperations [](start = 24, length = 22)

This file is `DataLoadSaveCatalog.cs` which before your change held `DataLoadSaveOperations` and which now holds `DataOperations`. Perhaps renaming the file before you check in would be appropriate.

Reply:
Will do

---
In reply to: [231714321](https://github.com/dotnet/machinelearning/pull/1569#discussion_r231714321) [](ancestors = 231714321)